src/getCurrentPosition/getCurrentPosition.types.ts

```diff
@@ -11,7 +11,23 @@ export type GeolocationSuccessCallback = (position: {
     timestamp: number;
 }) => void;
 
-export type GeolocationErrorCallback = (error: {code: number; message: string; PERMISSION_DENIED: number; POSITION_UNAVAILABLE: number; TIMEOUT: number}) => void;
+export type GeolocationErrorCallback = (error: {
+    code: GeolocationErrorCode;
+    message: string;
+    PERMISSION_DENIED: GeolocationErrorCode.PERMISSION_DENIED;
+    POSITION_UNAVAILABLE: GeolocationErrorCode.POSITION_UNAVAILABLE;
+    TIMEOUT: GeolocationErrorCode.TIMEOUT;
+
+    /* Web only */
+    NOT_SUPPORTED?: GeolocationErrorCode.NOT_SUPPORTED;
+}) => void;
+
+export enum GeolocationErrorCode {
```

Comment:
we actually [decided not to use Enums ](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#enum-vs-union-type). I don't know if we also want to follow that rule in this library, but it might be a good idea to follow the same rule

I should have added that ESLint rule here, too 😓 

In this case, we can just do

```ts
const GeolocationErrorCode = {
    PERMISSION_DENIED = 1,
    POSITION_UNAVAILABLE = 2,
    TIMEOUT = 3,
    NOT_SUPPORTED = -1,
}
```